B/BinningAnalysis/Compat.toml

```diff
@@ -1,8 +1,11 @@
 ["0.1"]
 EllipsisNotation = "0.0-0.4"
 
-["0.1-0.2"]
+["0.1-0.2.0"]
 Lazy = "0.1-0.13"
 RecursiveArrayTools = "0.0-0.20"
 Reexport = "0.0-0.2"
 julia = "1.0-1.3"
+
+["0.2.1"]
+julia = "1.0-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.